src/components/molecules/OcBreadcrumb/OcBreadcrumb.vue

```diff
@@ -110,6 +131,12 @@ export default {
 
       return [...this.items].reverse()[0]
     },
+    contextMenuLabel() {
+      return this.$gettext("Show actions for current folder")
+    },
+    showContextmenu() {
```

Comment:
```suggestion
    showContextMenu() {
```
for consistency with the computed prop above.